docs/guides/advanced/ServerRendering.md

```diff
@@ -53,7 +53,7 @@ render(<Router history={history} routes={routes} />, mountNode)
 You need to do
 
 ```js
-match({ routes, location }, (error, redirectLocation, renderProps) => {
-  render(<Router {...renderProps} history={history} />, mountNode)
+match({ history, routes }, (error, redirectLocation, renderProps) => {
```

Comment:
This is actually the "what to do on the client when using server-side-rendering plus async routes" section.